### ISSUE TITLE
Missing content updates when updating and restarting NVDA after a browser is started

### ISSUE THREAD

**Issue by bramd** [nvaccess/nvda#8241]
### Steps to reproduce:

1. Start Chrome and/or Firefox and open some pages
2. Update NVDA and install the update so NVDA is restarted
3. Open https://github.com/nvaccess/nvda/issues/ and go to page 2

### Expected behavior:

Page 2 of issues is displayed e.g. content updates are represented in the virtual buffer.

### Actual behavior:

Updated content without a page reload is no longer represented in the virtual buffer. It is after pressing nvda+f5.

### System configuration:
NVDA version:
next-15103,1df70fee

NVDA Installed

Windows version:
Windows 10 Version 1803 (OS Build 17134.1)

Name and version of other software in use when reproducing the issue:

* Chrome Version 66.0.3359.139 (Official Build) (64-bit)
* Firefox Nightly

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
* I've observed this with previous updates from next versions as well

**Comment by bramd** [comment]
When closing a tab in Chrome in this state, I get the following in the log which might be related:

```
DEBUGWARNING - RPC process 4300 (chrome.exe) (13:16:00.476):
Thread 11268, build\x86_64\remote\inProcess.cpp, execInThread, 175:
Could not open UI thread

ERROR - RPC process 4300 (chrome.exe) (13:16:00.483):
Thread 11268, build\x86_64\vbufBase\backend.cpp, VBufBackend_t::terminate, 234:
Could not execute renderThread_terminate in UI thread
```

**Comment by zstanecic** [comment]
oops! i can reproduce that in the latest master with the latest chrome!


W dniu 07.05.2018 o 13:17, Bram Duvigneau pisze:

**Comment by Neurrone** [comment]
I've noticed this in Firefox ever since the fix landed to have it read firefox pages properly when restarting NVDA. This problem goes away after restarting the browser.

**Comment by Adriani90** [comment]
@bramd, @Neurrone, @gregjozk, @zstanecic can you still reproduce this issue in NVDA 2019.1.1 even after you run the com registration fixing tool in nvda menu / tools?

**Comment by Adriani90** [comment]
I am not able to reproduce it with Chrome 74 and Firefox 66.0.3.

**Comment by gregjozk** [comment]
Hello,

it is fixed in my opinion; I don't know, which ticket covers a
workable solution.

regards

2019-05-07 19:42 GMT+02.00, Adriani90 <notifications@github.com>:

**Comment by Adriani90** [comment]
Thanks for reporting. I am closing this as works for me. If any one still has this issue, please comment on it and we can reopen the discussion.
